src/views/jobManager/editJob/perfTab.ts

```diff
@@ -0,0 +1,340 @@
+        selected: options["prefetch"] === `true`,
+      },
+      {
+        value: `false`,
+        text: `Do not use prefetch`,
+        description: `False`,
+        selected: options["prefetch"] === `false`,
+      },
+    ])
+    .addInput(
+      `qaqqinilib`,
+      `qaqqinilib`,
+      `	Specifies a QAQQINI library name. Used to specify the library that contains the qaqqini file to use. A qaqqini file contains all of the attributes that can potentially impact the performance of the DB2 for IBM i database engine. You must have *JOBCTL special authority to use qaqqinilib.`,
+      { default: options["qaqqinilib"] || `` }
+    )
+    .addSelect(`query optimize goal`, `Query optimize goal`, [
+      {
+        value: `0`,
+        text: `Optimize query for first block of data (*ALLIO) when extended dynamic packages are used; Optimize query for entire result set (*FIRSTIO) when packages are not used`,
+        description: `0`,
+        selected: options["query optimize goal"] === `0`,
+      },
+      {
+        value: `1`,
+        text: `Optimize query for first block of data (*FIRSTIO)`,
+        description: `1`,
+        selected: options["query optimize goal"] === `1`,
+      },
+      {
+        value: `2`,
+        text: `Optimize query for entire result set (*ALLIO)`,
+        description: `2`,
+        selected: options["query optimize goal"] === `2`,
+      },
+    ])
+    .addSelect(`query timeout mechanism`, `Query timeout mechanism`, [
+      {
+        value: `cancel`,
+        text: `The queryTimeout feature uses a database CANCEL request to cancel a running SQL statement after the specified timeout expires`,
+        description: `Cancel`,
+        selected: options["query timeout mechanism"] === `cancel`,
+      },
+      {
+        value: `qqrytimlmt`,
+        text: `The queryTimeout feature uses the QQRYTIMLMT feature of the database engine.`,
+        description: `qqrytimlmt`,
+        selected: options["query timeout mechanism"] === `qqrytimlmt`,
+      },
+    ])
+    // TODO: the followng two properties use a system default value, which is not supported by the current version of the driver:
+    // .addInput(`recieve buffer size`, `Recieve buffer size`, `	Specifies the buffer size used to receive data through the socket connection between the front-end driver and the IBM i system.
+    //   NOTE: This does not specify the actual receive buffer size. It is only used as a hint by the underlying socket code.`, {default: options["recieve buffer size"] || ``})
+    // .addInput(`send buffer size`, `Send buffer size`, `Specifies the buffer size used to send data through the socket connection between the front-end driver and the IBM i system.
+    //   NOTE: This does not specify the actual send buffer size. It is only used as a hint by the underlying socket code.`,
+    //   {default: options["send buffer size"] || ``})
+    .addSelect(`variable field compression`, `Variable field compression`, [
```

Comment:
Missing the "use block update" property.
<html><body>
<!--StartFragment-->

"use block update" | This property allows the toolbox JDBC driver to use a block      update mode when inserting or updating blocks of data into the database.        This is an optimized version of the batch insert or batch update. | no | "true"       "false" | "false"
-- | -- | -- | -- | --


<!--EndFragment-->
</body>
</html>

Reply:
@davecharron I had a look into this but couldn't see it in the JDBC options docs online. Do you have docs for this?

https://www.ibm.com/docs/en/i/7.2?topic=ssw_ibm_i_72/rzahh/javadoc/com/ibm/as400/access/doc-files/JDBCProperties.htm

Reply:
My bad, I was looking at an internal link.  I'm not finding the property in any externally available docs.